foxpuppet/windows/browser/panel_ui/panel_ui.py

```diff
@@ -43,6 +40,20 @@ def create(
         panel_items.update(PANEL_ITEMS)
         return panel_items.get(_id, PanelUI)(window, root)
 
+    @property
+    def is_barged(self) -> bool:
```

Comment:
I can see where you get this name from but would a user understand what `barged` means? Honestly, if I didn't see the `get_attribute("barged")` I wouldn't know what it meant.